examples/SampleApp/App.tsx

```diff
@@ -96,7 +101,29 @@ const App = () => {
   );
 };
 
-const DrawerNavigator: React.FC<{
+const DrawerNavigator: React.FC = () => {
+  const { overlay } = useOverlayContext();
+
+  return (
+    <Drawer.Navigator
+      screenOptions={{
+        gestureEnabled: Platform.OS === 'ios' && overlay === 'none',
+      }}
+      drawerContent={(props) => <MenuDrawer {...props} />}
```

Comment:
?
```suggestion
      drawerContent={MenuDrawer}
```

Reply:
This apparently doesn't work because it's the drawer content, just checked it out.

https://reactnavigation.org/docs/drawer-navigator/#drawercontent

```
<Drawer.Navigator drawerContent={(props) => <CustomDrawerContent {...props} />}>
  {/* screens */}
</Drawer.Navigator>
```